Sources/Time+Foundation.swift

```diff
@@ -7,6 +7,7 @@
 //
 
 import Foundation
+import Dispatch
```

Comment:
Hi @rafaelGuerreiro! Thanks for your contribution. I think it would be a right decision to put `import Dispatch` inside `#if os(Linux)` directive. I would be really grateful if you’d update your PR.

Reply:
Hey @dreymonde, thanks for your review. I've updated it.